tests/device_test.py

```diff
@@ -857,13 +866,24 @@ def test_playing_status_no_media_legacy(self, mocked_requests_post):
         self.assertEqual("PLAYING", device.get_playing_status())
 
     @mock.patch('requests.post', side_effect=mocked_requests_post)
-    def test_volume(self, mocked_requests_post):
+    def test_get_volume(self, mocked_requests_post):
         device = self.create_device()
         self.assertEqual(-1, device.get_volume())
 
-        device.rendering_control_url = RENDERING_CONTROL_URL
+        device.rendering_control_url = RENDERING_CONTROL_URL_GET_VOLUME
         self.assertEqual(64, device.get_volume())
 
+    @mock.patch('requests.post', side_effect=mocked_requests_post)
+    def test_set_volume(self, mocked_requests_post):
+        device = self.create_device()
+
+        device.rendering_control_url = RENDERING_CONTROL_URL_SET_VOLUME
+        # self.assertEqual(False, device.set_volume(-10))
```

Comment:
Can you please remove these comments as well. 
And then run the base command again, because GitHub still steals me I can't rebase this onto master due to conflicts :/

Reply:
Done